config/env.yml.template

```diff
@@ -3,4 +3,5 @@ AK_PASSWORD: 'password'
 AK_USERNAME: 'username'
 AK_HOST: 'https://act.sumofus.org/'
 HPC_RULE_ID: '22'
-AK_SUBSCRIPTION_PAGE_NAME: 'registration'
\ No newline at end of file
+AK_SUBSCRIPTION_PAGE_NAME: 'registration'
+ACTIONS_CACHE_SIZE: 100
```

Comment:
To be more precise the env file that is used for running specs on CircleCI is `circle/test.env`.